source/includes/_capture_external_event_api.md

```diff
@@ -0,0 +1,72 @@
+# Capture External Event API
+
+Capture External Event API is a public ReachLocal external interface for vendors to use.
+
+## Authorization - Capture External Event API
```

Comment:
I feel that the "- Capture External Event API" portion could be omitted.  This subsection is under a section header titled Capture External Event API so this portion is redundant.

Reply:
I had to make 'Authorization' unique or else if you clicked it you were taken to the 'Authorization' link under Client Reporting. It's a limitation with Markdown. I'll shorten the part after the dash until there is a remedy for this.